docs/source/programming/SSH-SWC-cluster.md

```diff
@@ -0,0 +1,235 @@
+# Set up SSH for the SWC HPC cluster
+
+This guide explains how to connect to the SWC's HPC cluster via SSH.
+
+```{include} ../_static/swc-wiki-warning.md
+```
+
+```{include} ../_static/code-blocks-note.md
+```
+
+## Abbreviations
+| Acronym | Meaning |
+| --- | --- |
+| SWC | Sainsbury Wellcome Centre |
+| HPC | High Performance Computing |
+| SLURM | Simple Linux Utility for Resource Management |
+| SSH | Secure (Socket) Shell protocol |
+| IDE | Integrated Development Environment |
+| GUI | Graphical User Interface |
+
+## Prerequisites
+- You have a SWC account and know your username and password.
+- You have read the [SWC wiki's section on High Performance Computing (HPC)](https://wiki.ucl.ac.uk/display/SSC/High+Performance+Computing), especially the [Logging into the Cluster page](https://wiki.ucl.ac.uk/display/SSC/Logging+into+the+Cluster).
+- You have an SSH client installed on your computer. This is usually pre-installed on Linux and macOS, but not on Windows. If you are using Windows, we recommend installing [Git Bash](https://gitforwindows.org/).
+- You know the basics of using the command line, i.e. using the terminal (Git Bash on Windows) to navigate the file system and run commands.
+
+## Log into the cluster
+Run the following commands on the terminal (typing your `<SWC-PASSWORD>` both times when prompted):
```

Comment:
Might be worth mentioning the password will not show up (I remember been very confused by this the first time I logged into a HPC).